templates/student_shop.html

```
@@ -195,77 +176,66 @@ <h2 class="h5">No Items Available</h2>
+                            <strong>Expires:</strong> {{ entitlement.display_expiry_date }}
                         </p>
                         {% endif %}
 
-                        <div class="card-text text-muted flex-grow-1">{{ (entitlement.store_item.description or 'No description available.') | markdown }}</div>
+                        <div class="card-text text-muted flex-grow-1">{{ (entitlement.item_description or 'No description available.') | markdown }}</div>
 
                         <div class="mt-auto">
-                            {% if entitlement.store_item.item_type == 'hall_pass' %}
+                            {% if entitlement.is_hall_pass %}
                                 <!-- Hall Pass - Use immediately -->
                                 <button class="btn btn-success w-100 use-item-btn"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)